gtsam_unstable/dynamics/tests/testIMUSystem.cpp

```diff
@@ -8,7 +8,7 @@
 #include <CppUnitLite/TestHarness.h>
 
 #include <gtsam/slam/BetweenFactor.h>
-#include <gtsam/slam/PriorFactor.h>
+#include <gtsam/nonlinear/PriorFactor.h>
```

Comment:
Forgotten case?

Reply:
This file initializes a PriorFactor on line 49.